src/uct/base/uct_cm.h

```diff
@@ -35,10 +35,8 @@ UCS_CLASS_DECLARE(uct_listener_t, uct_cm_h);
 
 
 #define uct_cm_ep_peer_error(_cep, _fmt, ...) \
-   ({ \
-        uct_cm_t *cm_base = ucs_container_of((_cep).super.super.iface, uct_cm_t, iface); \
-        ucs_log(cm_base->config.failure_level, _fmt, ## __VA_ARGS__); \
-   })
+    uct_cm_t *cm_base = ucs_container_of((_cep).super.super.iface, uct_cm_t, iface); \
```

Comment:
need to add `{ }` since there is local var, also pls start cm_base with `_`, i.e. `_cm_base`